PusherMainView/PusherMainView/Base.lproj/Pusher.storyboard

```diff
@@ -11,24 +11,24 @@
             <objects>
                 <viewController storyboardIdentifier="PusherViewController" id="XfG-lQ-9wD" customClass="PusherViewController" customModule="PusherMainView" customModuleProvider="target" sceneMemberID="viewController">
                     <view key="view" translatesAutoresizingMaskIntoConstraints="NO" id="m2S-Jp-Qdl">
```

Comment:
Note: These value are auto-generated by Interface builder after removing live activity checkbox and adding the push type textfield.